api/views/assets.py

```diff
@@ -594,3 +596,22 @@ def perform_update(self, serializer):
                     )
                 }
             )
+
+
+class PoliceAbstract(APIView):
```

Comment:
Hey @nelmandela,

Kindly share your reason for adding this view seeing as I do not see it being used.

Reply:
its for authentication purposes, linking the uploaded file to logged in user and implementing file storage